presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -72,6 +72,7 @@
     private boolean exchangeCompressionEnabled;
     private boolean legacyArrayAgg;
     private boolean legacyOrderBy;
+    private boolean groupByUsesEqualTo;
```

Comment:
call it `legacyGroupByUsesEqualTo` . ditto for the config name.

Reply:
@haozhun had suggested deprecated.group-by-uses-equals, I can change this, let me know which one to use.

Reply:
@wenleix This flag is intended as a kill switch, not a legacy flag like the others.